### ISSUE TITLE
Multi-target unit for status - cast trigger

### ISSUE THREAD

**Issue by Causese** [WeakAuras/WeakAuras2#817]
**Is your feature request related to a problem? Please describe.**
1) On council type fights, when there's multiple bossX units, sometimes boss 1/2 swaps right on pull.
Also duplicating a cast trigger for each bossX unit is also an annoying process. 

2) For non boss units aka nameplate units it would also be useful to have a multi-target unit option 
(for dungeons specifically, since "status - cast" auras hide if the unit's cast is interrupted. An event - cleu trigger wouldn't hide for example)

**Describe the solution you'd like**
basically a multi-target unit for status - cast (unit)

**Additional context**
CLEU triggers used to be popular for this but imo status - cast is the new meta because you don't have to worry about setting the right timed duration etc since it's all done by blizzard (unitcastinginfo)
In raids the cast time often changes based on raid difficulty, real horror for event - cleu trigger.

**Comment by InfusOnWoW** [comment]
This is now implemented for the next release with a "Multi" option for unit. There's also a option to check the Caster's target, which dynamically updates wheneve the caster's target changes.
